src/test/java/ua/yakaboo/BaseTests.java

```diff
@@ -0,0 +1,126 @@
+package ua.yakaboo;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BaseTests extends WebDriverSettings {
```

Comment:
Please, rename your `WebDriverSettings ` class to `BaseTest`, and `BaseTests` to smth like `CommonTests`

Reply:
Renamed